addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -60,25 +60,25 @@
     <xpath expr="//div[hasclass('o_wevent_registration_single_select')]" position="before">
         <div class="flex-md-grow-1 mt-1 me-2 text-end">
             <t t-if="tickets.price">
-                <t t-if="(tickets.price - tickets.price_reduce) &gt;0">
+                <t t-if="(tickets.total_price - tickets.total_price_reduce) &gt;0">
                     <del t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'"
                          class="text-danger me-1"
-                         t-field="tickets.price"
+                         t-field="tickets.total_price"
                          t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.currency_id}"/>
                     <del t-else=""
                          class="text-danger me-1"
-                         t-field="tickets.price_incl"
+                         t-field="tickets.total_price_incl"
                          t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.currency_id}"/>
                 </t>
                 <span t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'"
                       class="badge text-bg-secondary fs-6"
-                      t-field="tickets.price_reduce"
+                      t-field="tickets.total_price_reduce"
                       t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.currency_id}"/>
                 <span t-else=""
                       class="badge text-bg-secondary fs-6"
-                      t-field="tickets.price_reduce_taxinc"
```

Comment:
It seems like "price_reduce_taxinc" only had these 2 usage, can we remove the field then?
Please check for other fields as well.

Reply:
indeed good catch